StPetersburg/InEtn66217.xml

```diff
@@ -0,0 +1,231 @@
+                  </decoDesc>
+                  <additions>
+                    <list>
+                      <item xml:id="a1">
+                        <locus target="#7r"></locus>
+                        <desc type="GuestText">A table of shadows length during daytime in different months.</desc>
+                        <q xml:lang="gez"></q>
+                      </item>
+                      <item xml:id="a2">
+                        <locus target="#7v"></locus>
+                        <desc type="GuestText"><title type="incomplete" ref="LIT1834Mahlet"></title></desc>
+                        <note>18 lines of the text.</note>
+                        <q xml:id="gez"></q>
+                      </item>
+                      <item xml:id="a3">
+                        <locus from="63r" to="64r"></locus>
+                        <desc type="GuestText"></desc>
+                        <note>The quoted is the incipit.</note>
+                        <q xml:lang="gez">መንፈሶሙ፡ ነደ፡ እሳት፡ እሙንቱ፡ ወቅዱሳን፡ ኪሩቤል፡ ወሱራፌል፡ እለ፡ ፮ ክነፊሆሙ፡ ኪያከ፡ ይሴብሑ፡ በዓቢይ፡ ቃል፡ ወይዜምሩ። በቃል፡ ዘኢያርምም፡ ወይብሉ፡ ቅዱስ፡ ቅዱስ፡ ቅዱስ፡ እግዚአብሔር፡
+                        ፀባዖት፡</q>
+                      </item>
+                    <item xml:id="a4">
+                      <locus target="#64r'"></locus>
+                      <desc type="MagicText">Magic text against enemies with application instructions written in Amharic</desc>
+                      <note>The quoted is the incipit</note>
+                      <q xml:lang="gez">በርሃ፡ ይእቲ፡ ከሪ፡ ይሪ፡ ጠው፡ ራነ፡ በጀሊን፡ <gap reason="ellipsis"></gap> ለእለ፡ መጽኡ፡ እምሥራቅ፡ ወለእለ፡ መጽኡ፡ እምዕራብ፡ ለእለ፡ መጽኡ፡ እምሰሜን፡
+                    ወለእለ፡ መጽኡ፡ እምደቡብ፡ ድፍን፡ ልቦሙ፡ ወዝርዝር፡ ምክሮሙ፡ ወዕፁ፡ ጕረኤሆሙ፡ ለአጽራርየ፡</q>
+                    </item>
+
+                    </list>
+                    <list>
+                      <item xml:id="e1">
+                        <locus target="#49r"></locus>
+                        <desc>The name of the woner and the copyist are erased.</desc>
+                      </item>
+                    </list>
+                  </additions>
+
+                  <bindingDesc>
+                     <binding contemporary="true" xml:id="binding">
+                        <decoNote xml:id="b1">Two wooden boards. Both boards are cracked and repaired with a piece of thick thread.</decoNote>
+                        <decoNote xml:id="b2" type="bindingMaterial">
+                          <material key="wood"/></decoNote>
+                     </binding>
+                  </bindingDesc>
+
+               </physDesc>
+               <history>
+                 <provenance>The manuscript was purchased by <persName ref="PRS12886Balashova"></persName> in <placeName ref="LOC3010Ethiop"></placeName> with money
+                 provided by <persName ref="PRS7649Olderogg"></persName>.</provenance>
+                  <acquisition>The manuscript was granted by <persName ref="PRS7649Olderogg"></persName> to the <placeName ref="INS0524InEtn"></placeName>
```

Comment:
donated or bequeathed? "Granted" seems unusual to me, but I'm not an expert

Reply:
done

Reply:
I would say "donated" (i do not think "granted" is associated with this situation); "bequeathed" is after the owner's death (hope she is still fine). Is the year of donation known?

Reply:
already done